src/Drawer/index.js

```diff
@@ -128,30 +114,34 @@ const Drawer = ({
                 },
               ])}
               onClick={isHorizontal ? onNext : onPrev}
+              aria-controls={panelId}
+              aria-label={isHorizontal ? "Next" : "Previous"}
```

Comment:
I'm pretty sure we can use some flex direction magic to solve for the `isHorizontal` checks, but it seems way out of scope for this one PR